docs/user-guide/building-assistants.rst

```diff
@@ -804,6 +889,31 @@ Then you can retrain your bot and test it again:
 
 Don’t forget to add a few end to end stories to your ``test_stories.md`` for testing as well.
 
+
+.. note::
+    Here's a minimal checklist of  of files we modified to handle unexpected user input:
+
+      - ``actions.py``: 
+          - define ``action_greet``
+      - ``data/nlu.md``: 
+          - add training data for an ``explain`` intent
+      - ``domain.yml``:
+          - specify that intent ``greet`` should trigger ``action_greet_user`` in the ``intents`` section
+          - add ``action_greet_user`` to the ``actions`` section
+          - set the type of ``requested_slot`` to categorical 
+              - add all the required slots from the ``sales_form`` as values
+          - add the ``explain`` intent
+          - add responses for contextual question interruptions (``explain``)
+      - ``data/stories.md``: 
+          - remove stories using mapped intents if you have them
+          - add stories with FAQ interruptions in the middle of filling a form
+          - add stories with contextual question interruptions  (``explain``) in the middle of filling a form
+              - specify the value of ``requested_slot`` before the interruption
+              - the response to the interruption should match the value of ``requested_slot`` 
```

Comment:
this checklist seems really long and complex... is there any way we could make it a bit more concise?

Reply:
how about now?

Reply:
how about now?